src/components/icon.cjsx

```diff
@@ -24,6 +24,7 @@ module.exports = React.createClass
   render: ->
     classNames = classnames('tutor-icon', 'fa', "fa-#{@props.type}", @props.className, {
       'fa-spin': @props.spin
+      'clickable': @props.tooltip and @props.tooltipProps.trigger is 'click'
```

Comment:
woah, pretty slick solution! I wonder how that affects other places where we use `<Icon>`...